### ISSUE TITLE
Tell `cclambda` When to Quit

### ISSUE THREAD

**Issue by JonathonMisiewicz** [psi4/psi4#2521]
Currently, if `cclambda` fails to converge, it will continue with the computation, with several zeros that shouldn't be there. It should _at the very least_ warn the user that it did not converge the roots, and not print data as if it did converge the roots. I would expect it would quit the computation immediately.

Fix #943 first, please.

**Comment by lothian** [comment]
I agree that it should warn the user more clearly, but the lack-of-exit behavior is intentional.  It's an attempt to salvage useful data rather than simply quitting and giving nothing from a potentially very long excited-state calculation.

**Comment by loriab** [comment]
We were supposing it was something like that. Would it be reasonable to print results only from converged roots and then ending with coffee instead of beer? That could avoid the banks of zeros and the quick impression that all's well.

**Comment by lothian** [comment]
Yes, absolutely.
